api.swagger.yaml

```diff
@@ -2592,13 +2592,37 @@ paths:
         - 'blogs'
       description: 'Search for blogs. An empty array is returned if no blogs where found.'
       parameters:
+        - name: title
+          description: 'Filter on the blog title. Matches on whole words.'
+          in: query
+          schema:
+            type: array
+            items:
+              type: string
+              example: 'Guidelines for creating a strong password'
+        - name: title_autocomplete
+          description: 'Autocomplete term. Matches on blog name, per letter, from the start of the word.'
```

Comment:
Can this be done using the existing data in solr? If not, then I think we should do it selectively per CT when needed as Edge NGram can be pretty heavy on the indexing side

If it can already be done for all these CTs without adding more data into solr then ignore me

Reply:
Using existing solr properties. Doing a wildcard filter query. So a bit of a hack, not using solr Q but it seems to work.